packages/amazonq/package.json

```diff
@@ -1322,26 +1323,40 @@
                     "fontCharacter": "\\f1de"
                 }
             },
-            "aws-schemas-registry": {
+            "aws-sagemaker-code-editor": {
```

Comment:
I think you'll have to rebase to remove these changes. Solved this issue with Laura, so she is an expert now :)